enterprise/j2ee.dd.webservice/src/org/netbeans/modules/j2ee/dd/impl/resources/xml.xsd

```diff
@@ -0,0 +1,81 @@
+<?xml version='1.0'?>
```

Comment:
It would be great if you could add the license info from the W3C to the W3C files. It makes it easier to reason about the copyright. See:
`enterprise/web.core/src/org/netbeans/modules/web/taglib/resources/xml.xsd`
for a sample.